bin/gnng

```diff
@@ -369,19 +293,21 @@ def handle_output(all_rows, opts):
             print 'DEVICE: {}'.format(dev)
             print_table(rows)
 
-def print_table(rows, labels=None, has_header=True, separate_rows=False,
-                wrapfunc=None, delim=' | ', wrap_last=False):
+def print_table(rows, labels=None):
     """
     Print the interface table for a device
     """
     if labels is None:
         labels = ROW_LABELS
-    if wrapfunc is None:
-        wrapfunc = lambda x: wrap_onspace(x, 20)
 
-    print indent([labels] + rows, hasHeader=has_header,
-                 separateRows=separate_rows, wrapfunc=wrapfunc, delim=delim,
-                 wraplast=wrap_last)
+    output_table = PrettyTable()
+    output_table.field_names = labels
```

Comment:
Looking great. I just have a couple things to make the display of the table look a little cleaner.

Currently on a test device, the output looks like this:

```bash
$ bin/gnng arista-sw1
DEVICE: arista-sw1
+-------------+----------------+-----------------+---------+----------+--------------+
|  Interface  |   Addresses    |     Subnets     | ACLs IN | ACLs OUT | Description  |
+-------------+----------------+-----------------+---------+----------+--------------+
|  Ethernet1  | 172.16.118.136 | 172.16.118.0/24 |         |          | TO_UBUNTU_VM |
| Management1 | 192.168.56.203 | 192.168.56.0/24 |         |          |              |
+-------------+----------------+-----------------+---------+----------+--------------+
```

1. Change the import to `import prettytable` so that we can reference the next items.
2. Add these lines:

```python
    output_table.align = 'l'  # Left align data
    output_table.vrules = prettytable.prettytable.ALL
    output_table.hrules = prettytable.prettytable.HEADER
```

Then the table looks like this:

```bash
$ bin/gnng arista-sw1
DEVICE: arista-sw1
| Interface   | Addresses      | Subnets         | ACLs IN | ACLs OUT | Description  |
+-------------+----------------+-----------------+---------+----------+--------------+
| Ethernet1   | 172.16.118.136 | 172.16.118.0/24 |         |          | TO_UBUNTU_VM |
| Management1 | 192.168.56.203 | 192.168.56.0/24 |         |          |              |
```